src/backend/aspen/test_infra/models/pathogen_repo_config.py

```diff
@@ -1,35 +1,65 @@
+import random
+from typing import Dict, Optional, Tuple
+
 from aspen.database.models import Pathogen, PathogenRepoConfig, PublicRepository
 from aspen.database.models.usergroup import generate_random_id
 from aspen.test_infra.models.pathogen import random_pathogen_factory
+from aspen.util.split import SplitClient
 
 
-def setup_gisaid_and_genbank_repo_configs(
+def pathogen_repo_config_factory(prefix, pathogen, public_repository):
+    pathogen_repo_config = PathogenRepoConfig(
+        prefix=prefix, pathogen=pathogen, public_repository=public_repository
+    )
+
+    return pathogen_repo_config
+
+
+def setup_random_repo_configs(
```

Comment:
By default, creates a pathogen, two pathogen repos, and sets a split feature flag to determine which one is the default.